moveit_core/package.xml

```diff
@@ -25,12 +25,12 @@
   <build_depend>eigen_conversions</build_depend>
   <build_depend>eigen_stl_containers</build_depend>
   <build_depend>libfcl-dev</build_depend>
-  <build_depend version_gte="0.5.1">geometric_shapes</build_depend>
+  <build_depend version_gte="0.5.2">geometric_shapes</build_depend>
```

Comment:
Sure, but you could extend that argument to all of MoveIt's many dependencies - why do we need to explicitly specify it for this package? I thought the assumption is always *if its missing an API call, get the latest version*